server/fusor_server.gemspec

```diff
@@ -22,6 +22,9 @@ Gem::Specification.new do |s|
   s.add_dependency "net-ssh", '2.9.2' # last ruby 1.9 version
   s.add_dependency 'net-scp'
   s.add_dependency "sys-filesystem"
+  s.add_dependency "foreman_discovery"
+  s.add_dependency "foretello_api_v21"
+  s.add_dependency "egon"
```

Comment:
Should we be adding these here, or in the rpm spec (looks like discovery is missing of the three), or both to also ensure the necessary packages get installed in an rpm env?